README.md

```diff
@@ -204,9 +205,18 @@ Run the following from your app's top-level directory:
 bundle exec roo_on_rails
 ```
 
-You will (currently) need to have admin privileges on the
-`roo-dd-bridge-production` application for this to work. This will be addressed
-eventually.
+That command will sequentially run a number of checks. For it to run successfully, you will need:
+
+- a GitHub API token that can read your GitHub repository's settings placed in `~/.roo_on_rails/github-token`
+- the Heroku toolbelt installed and logged in
+- admin privileges on the `roo-dd-bridge-production` (this will be addressed eventually)
+- checks are run sequentially for staging and then for production. The process halts at any non-fixable failing check. To process only specific environments, you can set a config variable while running the command, like so:
+
+```
+ROO_ON_RAILS_ENVIRONMENTS=staging,production bundle exec roo_on_rails $(: this is the default behaviour)
```

Comment:
I think the `$(: ... )` syntax might confuse non-bash-expert readers. Maybe just use `#`-comments?